subsys/net/lib/mqtt/mqtt_transport.c

```diff
@@ -29,6 +29,11 @@ extern int mqtt_client_tls_read(struct mqtt_client *client, u8_t *data,
 extern int mqtt_client_tls_disconnect(struct mqtt_client *client);
 #endif /* CONFIG_MQTT_LIB_TLS */
 
+#if IS_ENABLED(CONFIG_MQTT_LIB_SOCKS)
```

Comment:
IS_ENABLED is to be used only with code (at least that's the only usage we have), besides that: use #ifdef or #if defined()

Sorry if my comment about using IS_ENABLED was unclear.

Reply:
ok, makes sense, thanks!

Reply:
Fixed now.